README.md

```diff
@@ -76,6 +76,8 @@ We maintain two distinct Docker setups for development and production environmen
 
 To build and run the Docker containers, execute the following commands from within the respective directory:
 
+We also have created a script to fix the configurations for you. You can find it in `docker/compose/dev/build/modify-configs.sh`. This script will look at your `.env.oid4vci-demo-frontend` to select an ecosystem (defaults to `sphereon`) and then looks at your docker-compose file for `DEMO_HOST_ADDRESS` (defaults to `localhost`) if not found. After that, it will copy the desired ecosystem agent's config files to a folder called `docker+epoch_date` like: `docker-1700059590` uses that as the configuration for the demo.
```

Comment:
```suggestion
We also have created a script to modify the configurations for you. You can find it in `docker/compose/dev/build/modify-configs.sh`. This script will look at your `.env.oid4vci-demo-frontend` to select an ecosystem (defaults to `sphereon`) and then looks at your docker-compose file for `DEMO_HOST_ADDRESS` (defaults to `localhost`) if not found. After that, it will copy the desired ecosystem agent's config files to a folder called `docker+epoch_date` like: `docker-1700059590` uses that as the configuration for the demo.
```

Reply:
Not sure I like the epoch approach, because people will probably call it multiple times, resulting in multiple folders and confusion, but let's leave it for now

Reply:
Also we should warn on not using localhost. So the default is not really we should allow to begin with. The demo simply will not work cross-device when using localhost, so no sense in using that as default

Reply:
what should I put if nothing is provided by the user? should I go to `OID4VP_AGENT_BASE_URI` which should be present in the agent's env?

Reply:
We should throw an error IMO pointing the user to the documentation and that the value needs to be provided and should reflect an external reachable IP address